os/arch/arm/src/stm32l4/stm32l4_idle.c

```diff
@@ -67,6 +75,70 @@
 #endif
 
 #define PM_IDLE_DOMAIN 0 /* Revisit */
+#define CONFIG_TICKLESS_TIME_CORRECTION
```

Comment:
Is there any special reason that this config is defined here?
If like this, you don't need to check this is defined or not in this file.(e.g line 162)

Reply:
TICKLESS_TIME_CORRECTION can be removed. I will check this and update.